flixel/addons/effects/FlxEffectSprite.hx

```diff
@@ -0,0 +1,134 @@
+package flixel.addons.effects;
+
+import flixel.FlxCamera;
+import flixel.FlxSprite;
+import flixel.graphics.FlxGraphic;
+import flixel.graphics.frames.FlxFrame;
+import flixel.math.FlxPoint;
+import flixel.math.FlxRect;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.display.Graphics;
+import openfl.geom.Point;
+
+/**
+ * Use this class to chain bitmap effects to your target FlxSprite frame.
+ * @author adrianulima
+ */
+class FlxEffectSprite extends FlxSprite
+{
+	/**
+	 * Enables and disables the effects update loop.
```

Comment:
Besides, is it really necessary to have this flag? There are already active and visible to control whether update and raw are run respectively.